dlc_developer_config.toml

```diff
@@ -23,7 +23,7 @@ huggingface_trcomp_mode = false
 trcomp_mode = false
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
```

Comment:
Remember to set `graviton_mode` to True to run graviton tests after we are finished with the x86 tests.

Reply:
running this build with off to build x86 images. In the next commit i will enable to build graviton images and then set `do_build` to false [here](https://github.com/aws/deep-learning-containers/blob/master/dlc_developer_config.toml#L41)

This will reuse images for subsequent runs.